ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersTable.vue

```diff
@@ -512,17 +509,13 @@ export default defineComponent({
           return props.validateTransfer && props.isMhrTransfer && !hasUnsavedChanges.value
         }
 
-        const groups = getTransferOrRegistrationHomeOwnerGroups()
-
         return ((props.validateTransfer || (!props.isMhrTransfer && localState.reviewedOwners)) &&
           (
             !hasMinimumGroups() ||
-            hasEmptyGroup.value ||
             (props.isMhrTransfer && !hasUnsavedChanges.value) ||
             !localState.isValidAllocation ||
             localState.hasGroupsWithNoOwners ||
-            (!localState.isUngroupedTenancy && hasUndefinedGroupInterest(groups)) ||
```

Comment:
isUngroupedTenancy checked that the tenancy type is sole owner or joints tenants, however we can have an N/A tenancy type, and no explicit groups. Under the condition above we will get an error because the group interest will be undefined.

Reply:
What about `hasUndefinedGroupInterest`? Where did that go?

Reply:
isValidAllocation, if we have groups checks, for total allocation error, which cover `hasUndefiendedGroupInterest`

![image](https://github.com/bcgov/ppr/assets/77707952/f672dea1-8133-4a10-b867-e00fffed7de9)

![image](https://github.com/bcgov/ppr/assets/77707952/b191b34e-6ebc-4fee-a3e7-41074afe30e7)

Reply:
ok, got it, thanks